lbrynet/daemon/Daemon.py

```diff
@@ -293,6 +296,23 @@ def _update_connection_status(self):
         if not self.connected_to_internet:
             self.connection_status_code = CONNECTION_STATUS_NETWORK
 
+    @defer.inlineCallbacks
+    def _auto_renew(self):
+        # automatically renew claims
+        # auto renew is turned off if 0 or some negative number
+        if self.auto_renew_claim_height_delta < 1:
+            return
```

Comment:
this will raise a deferred generator error (should be `defer.returnValue(None)`)

Reply:
fixed